projects/tasks.py

```diff
@@ -55,27 +55,25 @@ def end_daypasses():
             )
             invitation.status = Invitation.STATUS_BEYOND_DURATION
             invitation.save()
-        except Exception:
-            LOG.error(f"Error ending daypass invite {invitation.id}")
 
-        try:
-            # If this invite was on a day pass
-            day_pass_request = DayPassRequest.objects.get(invitation=invitation)
-            # And there are 10 requests on that same artifact
-            approved_requests = (
-                DayPassRequest.objects.all()
-                .filter(
-                    artifact=day_pass_request.artifact,
-                    status=DayPassRequest.STATUS_APPROVED,
-                    invitation__status=Invitation.STATUS_BEYOND_DURATION,
+            try:
+                day_pass_request = DayPassRequest.objects.get(invitation=invitation)
+                approved_requests = (
+                    DayPassRequest.objects.all()
+                    .filter(
+                        artifact=day_pass_request.artifact,
+                        status=DayPassRequest.STATUS_APPROVED,
+                        invitation__status=Invitation.STATUS_BEYOND_DURATION,
+                    )
+                    .count()
                 )
-                .count()
-            )
-            if approved_requests == 10:
-                # Send an email
-                handle_too_many_day_pass_users(day_pass_request.artifact)
-        except DayPassRequest.DoesNotExist:
-            pass
+                if approved_requests == settings.DAY_PASS_LIMIT:
+                    # Send an email
+                    handle_too_many_day_pass_users(day_pass_request.artifact)
+            except DayPassRequest.DoesNotExist:
+                pass
```

Comment:
Does this indicate that something is wrong, or that something unexpected is going on? Since we get the invitations from `get_invitations_beyond_duration`, this should mean it's a valid invitation, each of which should have one associated day pass request, right? Should it somehow be possible for an invitation to not have an associated day pass request?

If this state is not expected, I think it would be good to log something here.